content/en/docs/Installing and Configuring DreamFactory/_index.md

```diff
@@ -211,6 +211,18 @@ Immediately following this prompt you'll be informed of successful setup:
 	* your instance using a browser. You can run "php artisan serve"
 	* to try out your instance without setting up a web server.
 	**********************************************************
+After creating an Admin, your instance may show the following error:
+```
+Admins
+There is no root administrator selected. Some functionality might not work. Use df:root_admin command to choose one.
+```
+
+A very simple approach to solve it, is to reset the root from inside your container:
+`$ php artisan df:root`
+`$ php artisan migrate:fresh --seed`
+`$ php artisan cache:clear`
+
+Next, go to you localhost (backend) and create new credentials. This will assure your new admin details are updated and run a GOLD license.
```

Comment:
A couple things
We as developers have a localhost(backend). The customer will not. They will run dreamfactory on their own server with its own domain.

I think you only had to create new credentials because you ran migrate:fresh -- seed and dropped all the tables. Again we shouldnt be telling the user to do this. and if they dont just running php artisan df:root_admin should work

Also note that we have silver as well as gold.